src/azul/queues.py

```diff
@@ -430,19 +433,26 @@ def manage_lambdas(self, queues: Mapping[str, Queue], enable: bool):
             def submit(f, *args, **kwargs):
                 futures.append(tpe.submit(f, *args, **kwargs))
 
+            indexer = load_app_module('indexer')
+            lambdas = [
+                indexer.contribute,
+                indexer.contribute_retry,
+                indexer.aggregate,
+                indexer.aggregate_retry
+            ]
+            queues_to_lambda = {lambda_.queue: lambda_ for lambda_ in lambdas}
             for queue_name, queue in queues.items():
                 if queue_name == config.notifications_queue_name():
+                    # Since the indexer lambda writes to the notifications
```

Comment:
Couldn't we implement this exception more elegantly by patching `queues_to_lambda`?